app/controllers/public/sessions.js

```diff
@@ -0,0 +1,33 @@
+import Ember from 'ember';
+import { pick, last, chain } from 'lodash';
+
+const { Controller, computed } = Ember;
+
+export default Controller.extend({
+  sessions: [{ shortAbstract: 'An introduction to the event', id: '1', title: 'Welcome to FOSSASIA', startAt: new Date(), endAt: new Date(), track: { color: 'green', fontColor: 'green', id: 1, name: 'Track 1' }, microlocation: { id: 2, name: 'Room 1' }, speakers: [{ shortBiography: 'Works for ORG 1', id: 0, city: 'Delhi', name: 'Arnold Singh', speakingExperience: 'GSOC 2015', organisation: 'ORG 1', longBiography: '', photo: { 'iconImageUrl': '' } }] },
+  { shortAbstract: 'Welcome to open source', id: '2', title: 'Introduction to Open source', startAt: new Date(), endAt: new Date(), track: { color: 'red', fontColor: 'white', id: 2, name: 'Track 2' }, microlocation: { id: 3, name: 'Room 5' }, speakers: [{ shortBiography: '', id: 0, city: 'Delhi', name: 'Tux', speakingExperience: 'Speaker at the high conference', organisation: 'ORG 2', longBiography: '', photo: { 'iconImageUrl': '' } }] }
+  ],
+  sessionsByTracks: computed('sessions.[]', 'selectedTrackId', function() {
+    var allSessions = chain(this.get('sessions'))
+      .orderBy(['startAt', 'startAt'])
+      .groupBy('track.id')
+      .mapValues(function(v) {
```

Comment:
@geekyd that's exactly what is being done here too https://github.com/fossasia/open-event-frontend/pull/68#discussion_r117942196 but with one less operation. 

With one or two sessions, a single less operation isn't gonna matter. But with hundreds of session, a single operation, especially ones like `mapValues` matter a lot